plugins/reporters/opossum/src/test/kotlin/OpossumReporterTest.kt

```diff
@@ -213,18 +201,18 @@ class OpossumReporterTest : WordSpec({
 
         "create issues containing all issues" {
             val issuesFromFirstPackage =
-                opossumInput.getSignalsForFile("/pom.xml/compile/first-package-group/first-package@0.0.1")
+                opossumInput.getSignalsForFile("/pom.xml/compile/first-package-group/first-package@0.0.1/")
```

Comment:
I guess the trailing slash is not needed and could be removed again to reduce the diff?

Reply:
Could it make sense then to make the keys in `resourcesToAttribution` `File`s instead of `String` to be slash-agnostic and also have no forward- vs backward-slash issues between Linus and Windows?

Reply:
I don't think so. The paths are used in OpossumUI to build for example a browser for the file tree. I think it relies on forward slashes. So we are not using the to interact with the file system.
Also, we seem to work with the assumption that every path without children, represents a file (see `isFile` in `OpossumResources`). Something we can only know after building the entire tree.
So there is not much to be gained here from using `File`s I think.